exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,24 @@
+# Otlptrace Exporter
+
+To constructs a new Otlptrace Exporter and starts it:
+```
+exp, err := otlptrace.New(ctx, opts...)
+```
+# Otlptrace Client
+Otlptrace package define a trace exporter that uses a `otlptrace.Client` .
+
+`otlptrace.Client` manages connections to the collector, handles the transformation of data into wire format, 
+and the transmission of that data to the collector.
+
+# Otlptracegrpc and Otlptracehttp
+The otlptracegrpc and otlptracehttp implements a gRPC `otlptrace.Client` and 
+an HTTP `otlptrace.Client`respectively,
+both offering convenience functions .
```

Comment:
This for sure would not pass the markdownlint.
I personally use https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint

Reply:
I am so glad to receive all your suggestion.
I have already updated my PR, please review it.
Thnak u~